src/pages/partners/deriv-prime/why-deriv-prime/data.ts

```diff
@@ -0,0 +1,38 @@
+import { TString } from 'types/generics'
+
+type CardTypes = {
+    title?: TString
```

Comment:
Make a separate file for types inside `features->pages->partners->deriv-prime->why-deriv-prime->types.ts`